cpp/ps0000/main (1).cpp

```diff
@@ -1,11 +1,12 @@
-#inline <iostream>
+#include <iostream> // 2/2
```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`